vsdownload.py

```diff
@@ -55,21 +55,30 @@ def getArgsParser():
     parser.add_argument("--keep-unpack", const=True, action="store_const", help="Keep the unpacked files that aren't otherwise selected as needed output")
     parser.add_argument("--msvc-version", metavar="version", help="Install a specific MSVC toolchain version")
     parser.add_argument("--sdk-version", metavar="version", help="Install a specific Windows SDK version")
+    parser.add_argument("--architecture", metavar="arch", help="Target architecture to include (x86, x64, arm, arm64), defaults to host", nargs="+")
     parser.add_argument("--with-wdk-installers", metavar="dir", help="Install Windows Driver Kit using the provided MSI installers")
     parser.add_argument("--host-arch", metavar="arch", choices=["x86", "x64", "arm64"], help="Specify the host architecture of packages to install")
     parser.add_argument("--only-host", default=True, const=True, action="store_const", help="Only download packages that match host arch")
     return parser
 
 def setPackageSelectionMSVC16(args, packages, userversion, sdk, toolversion, defaultPackages):
     if findPackage(packages, "Microsoft.VisualStudio.Component.VC." + toolversion + ".x86.x64", warn=False):
+        if "x86" in args.architecture or "x64" in args.architecture:
+            args.package.append("Microsoft.VisualStudio.Component.VC." + toolversion + ".x86.x64")
+            args.package.append("Microsoft.VC." + toolversion + ".ASAN.X86")
```

Comment:
It looks like this is an addition to what packages we installed before, but I guess it's a very nice addition. Would be good to mention it explicitly though, that this also changes the sets of packages that gets installed (or single out that change into a quick PR before or after this one).